recipes/src/main/resources/quarkus-updates/io.quarkiverse.minio/quarkus-minio/3.8.yaml

```
@@ -1,5 +1,5 @@
 type: specs.openrewrite.org/v1beta/recipe
-name: io.quakus.updates.minio.minio38.UpdateAll
+name: io.quarkus.updates.minio.minio38.UpdateAll
 recipeList:
```

Might be good to also provide a `displayName` and `description`, as those get shown in the docs.
```suggestion
displayName: Update Minio client extension to 3.8
description: Update properties and adjust url property value for Minio 3.8.
recipeList:
```